Added Crowd Authentication plugin references

mgambini opened the pull request:
Hi,
has anyone checked the plugin?

varshavaradarajan commented:
Hi, 
will pick it up today.

varshavaradarajan commented on data/authorization_plugins.yml:
```diff
@@ -21,4 +21,13 @@
+  author_url: http://www.ditechonline.it/
+  author_name: Di.Tech S.p.A.
+  contact_url: mailto:marco.gambini@ditechspa.it?subject=Info about Crowd Authentication Plugin
+  paid: false
```
There should be a releases_url. The idea is that users can just download the jar in server plugins external. Check the other yaml files. The reason the authorization plugins don't have the releases url is that its already bundled with go and users don't have to download it.

mgambini replied:
Do you mean I should commit the plugin on github.com (now source code is hosted on gitlab.com) and use github continuous integration?

varshavaradarajan replied:
@mgambini , you can use any CI tool and any SCM platform as long you you can provide a jar that can be downloadable by users.

varshavaradarajan commented:
@mgambini , checking the README, it feels like one has to edit the crowd.properties and then build a jar. An easier experience for the user would be to download the jar straightaway and edit this file. Please look at  https://github.com/ashwanthkumar/gocd-slack-build-notifier as to how you can source the conf file. Can you please elaborate on the properties that is intended to be configured? Its optional. But might be nice to have. Your call.

mgambini commented:
@varshavaradarajan, I moved source code to GitHub, activated CI and published a first release: https://github.com/mgambini/crowd-integration-plugin/releases

Also I updated the data/authorization_plugins.yml file adding releases_url info.
Sincerely,
Marco

mgambini commented:
@varshavaradarajan, have you picked up my changes?
Cheers,
Marco
